synapse/models/person.py

```diff
@@ -1,41 +1,238 @@
+                    # ('seen:min', {'ptype': 'time:min',
+                    #               'doc': 'The earliest known time when the person owned or controlled the resource.'}),
+                    # ('seen:max', {'ptype': 'time:max',
+                    #               'doc': 'The most recent known time when the person owned or controlled the resource.'}),
+                )),
+                ('ps:persona:has', {}, (
+                    ('persona', ('ps:person', {}), {
+                        'ro': True,
+                        'doc': 'The persona who owns or controls the object or resource.',
+                    }),
+                    ('node', ('ndef', {}), {
+                        'ro': True,
+                        'doc': 'The object or resource that is owned or controlled by the persona.',
+                    }),
+                    ('node:form', ('str', {}), {
+                        'ro': True,
+                        'doc': 'The form of the object or resource that is owned or controlled by the persona.',
+                    }),
+                    # FIXME Add seen:min and seen:max
+                    # ('seen:min', {'ptype': 'time:min',
+                    #               'doc': 'The earliest known time when the person owned or controlled the resource.'}),
+                    # ('seen:max', {'ptype': 'time:max',
+                    #               'doc': 'The most recent known time when the person owned or controlled the resource.'}),
+                )),
+                # FIXME requires file:bytes
+                # ('ps:image', {}, (
+                #     ('person', ('ps:person', {}), {
+                #         'doc': '',
+                #     }),
+                #     ('image', ('file:bytes', {}), {
+                #         'doc': '',
+                #     }),
+                # # FIXME add an optional bounding box
+                # )),
+                ('ps:contact', {}, (
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The ou:org GUID which owns this contact.',
+                    }),
+                    ('asof', ('time', {}), {
+                        'date': 'The time this contact was created or modified.',
+                    }),
+                    ('person', ('ps:person', {}), {
+                        'doc': ' The ps:person GUID which owns this contact.',
+                    }),
+                    ('name', ('ps:name', {}), {
+                        'doc': 'The person name listed for the contact',
+                    }),
+                    ('title', ('str', {'lower': True, 'strip': True}), {
+                        'doc': 'The job/org title listed for this contact.',
+                    }),
+                    # FIXME requires file:bytes
+                    # ('photo', ('file:bytes', {}), {
+                    #     'doc': 'The photo listed for this contact.',
+                    # }),
+                    ('orgname', ('ou:name', {}), {
```

Comment:
This was a str:lwr before, but probably should have been ou:name

Reply:
I felt it made sense to use ou:name here

Reply:
agreed